driver-core/src/test/java/com/datastax/driver/core/ReplicationFactorTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.testng.annotations.Test;
+
```

Comment:
Is it possible to add an Integration test with a transient replica that runs only on > `OSS 4.0.0` and validates that the replication factor is parsed correctly?

Reply:
I should be able to do that.

Reply:
I think that would be welcome on 4.x branch as well. But you can do this in a follow-up ticket.

Reply:
Added: https://datastax-oss.atlassian.net/browse/JAVA-2746